### PR TITLE
Modify files for desktop version of mobile portfolio

### PR DESCRIPTION
**Pull request by tahseenzahra**
This branch contains:
- organized files of CSS
- desktop portfolio code
- transition to buttons

### CODE REVIEW

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @tahseenzahra :wave:,  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Well organized code and files
:heavy_check_mark: Professional documentation
:heavy_check_mark: Nice UI design

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly use the left alignment of the headline social media icons as in the figma design.
#### Figma
headline-works-section-figma
#### Impl
headline-works-section-impl

- [x] The contents of the project cards are aligned bottom (flex-end) in the figma design. Kindly rectify.
#### Figma
project-card-desk-figma
#### Impl
project-card-desk-impl

- [x] You should align left the languages, Frameworks and skills list as in the Figma design.
#### Figma
languages-section--figma

#### Impl
languages-section--impl


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -23,204 +30,207 @@
...
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
```
Comment (zieeco) [inline index.html:134]: **[REQUIRED CHANGES]**

- [ ] Hey, as required by the previous review, Please kindly align your list items to the left to be on same line with the paragraph and header content below: 

**YOUR DESIGN** :point_down: 
align

**FIGMA DESIGN** :point_down: 


figma

File: index.html [hunk 2]
```diff
@@ -232,12 +242,13 @@ <h3 class="mainText CreteRoundfont">About me</h3>
...
+      <div class="footer_contact">
+      <p class="item_1 Interfont footerTxt">
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
       <form
-        class="contactForm"
+        class="contactForm item_2"
         action="https://formspree.io/f/xeqnpapb"
         method="POST"
```
Comment (zieeco) [inline index.html:253]: - [ ] **[OPTIONAL]**
- Although the previous reviewer didn't point this out, however, consider them as **MANDATORY REQUIREMENT** for this project, try addressing them in your spare time..

- Kindly reduce the width of your paragraph and align the button to the left on the form
**YOUR DESIGN** :point_down: 

FORM


**FIGMA DESIGN** :point_down: 

FORMS

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @tahseenzahra 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by m05e5** [comment]
Hello

**Review by m05e5 — APPROVED** [review]
# Hi @tahseenzahra , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**